### DISCUSSION TITLE
How to manually add an AppImage

### DISCUSSION THREAD

**Discussion by m-delc** [TheAssassin/AppImageLauncher#568] [Q&A]
I have AppImages which, for whatever reason, are not recognized by AIL. How can I manually add them? Debian PC

**Comment by TheAssassin** [comment]
Some AppImages do not wish to be integrated. libappimage (and thus AppImageLauncher) respect that.

Which AppImages do you think should be integrated?

**Comment by TheAssassin** [comment]
In case you want to suggest a "force integration" feature, please don't hesitate to open a feature request issue.
